clients/python/src/fenlmagic/__init__.py

```diff
@@ -40,6 +45,7 @@ class FenlMagics(Magics):
     def __init__(self, shell, client):
         super(FenlMagics, self).__init__(shell)
         self.client = client
+        logger.info("fenlmagic extension loaded")
```

Comment:
maybe change to just `"extension loaded"` so that the output reads without the repeated "fenlmagic"?